### ISSUE TITLE
[Bug]: v8 Layer does not work with v9 Dialog/tabster

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#30545]
### Environment Information

- **Package version(s)**: latest v8, also latest v9/tabster (^6.0.1)

### Describe the issue:

Tabster applies `aria-hidden` to the layer host element that v8 Layers are appended to. The result is that any v8 pickers/callouts/comboboxes/dropdowns, etc. that are added to a v9 Dialog end up within an `aria-hidden` node and cannot be accessed with a screen reader.

The react-tabster `getParent` function looks for virtual parents, but the issue is that the Layer root node has a virtual parent, but not the layer host element. Since the layer host is the direct child of `<body>`, tabster only processes that node, and not the child Layers within it.

Layer host code is here: https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/Layer/Layer.base.tsx#L88-L111
react-tabster's getParent function is here: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-utilities/src/virtualParent/getParent.ts

It looks like we can't solve this using the custom `getParent` alone, since the layer host can't take on a single virtual parent (it could host multiple Layers with different virtual parents), and tabster never walks into the layer host to reach the individual layers. 

### If applicable, please provide a codepen repro:
https://codesandbox.io/p/sandbox/small-water-d72nwg?file=%2Fsrc%2Fexample.tsx

Products/sites affected: (if applicable)
This issue originated in an [ADO a11y bug](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/18519) impacting a partner team

**Comment by smhigley** [comment]
Closing, this was fixed in the merged PR.
